app/scripts/constants/sentry-state.ts

```diff
@@ -0,0 +1,416 @@
+    currentBlockGasLimit: true,
+    currentBlockGasLimitByChainId: true,
+  },
+  AddressBookController: {
+    addressBook: false,
+  },
+  AlertController: {
+    alertEnabledness: true,
+    unconnectedAccountAlertShownOrigins: false,
+    web3ShimUsageOrigins: false,
+  },
+  AnnouncementController: {
+    announcements: false,
+  },
+  AuthenticationController: {
+    isSignedIn: false,
+  },
+  NetworkOrderController: {
+    orderedNetworkList: [],
+  },
+  AccountOrderController: {
+    pinnedAccountList: [],
+    hiddenAccountList: [],
+  },
+  AppMetadataController: {
+    currentAppVersion: true,
+    currentMigrationVersion: true,
+    previousAppVersion: true,
+    previousMigrationVersion: true,
+    showTokenAutodetectModalOnUpgrade: false,
+  },
+  ApprovalController: {
+    approvalFlows: false,
+    pendingApprovals: false,
+    pendingApprovalCount: false,
+  },
+  AppStateController: {
+    browserEnvironment: true,
+    connectedStatusPopoverHasBeenShown: true,
+    currentPopupId: false,
+    onboardingDate: false,
+    currentExtensionPopupId: false,
+    defaultHomeActiveTabName: true,
+    fullScreenGasPollTokens: true,
+    hadAdvancedGasFeesSetPriorToMigration92_3: true,
+    nftsDetectionNoticeDismissed: true,
+    nftsDropdownState: true,
+    notificationGasPollTokens: true,
+    outdatedBrowserWarningLastShown: true,
+    popupGasPollTokens: true,
+    qrHardware: true,
+    recoveryPhraseReminderHasBeenShown: true,
+    recoveryPhraseReminderLastShown: true,
+    showBetaHeader: true,
+    showPermissionsTour: true,
+    showNetworkBanner: true,
+    showAccountBanner: true,
+    switchedNetworkDetails: false,
+    switchedNetworkNeverShowMessage: false,
+    showTestnetMessageInDropdown: true,
+    surveyLinkLastClickedOrClosed: true,
+    snapsInstallPrivacyWarningShown: true,
+    termsOfUseLastAgreed: true,
+    timeoutMinutes: true,
+    trezorModel: true,
+    usedNetworks: true,
+  },
+  MultichainBalancesController: {
+    balances: false,
+  },
+  BridgeController: {
+    bridgeState: {
+      bridgeFeatureFlags: {
+        extensionSupport: false,
+      },
+    },
+  },
+  CronjobController: {
+    jobs: false,
+  },
+  CurrencyController: {
+    currentCurrency: true,
+    currencyRates: true,
+  },
+  DecryptMessageController: {
+    unapprovedDecryptMsgs: false,
+    unapprovedDecryptMsgCount: true,
+  },
+  EncryptionPublicKeyController: {
+    unapprovedEncryptionPublicKeyMsgs: false,
+    unapprovedEncryptionPublicKeyMsgCount: true,
+  },
+  EnsController: {
+    ensResolutionsByAddress: false,
+    ensEntries: false,
+  },
+  GasFeeController: {
+    estimatedGasFeeTimeBounds: true,
+    gasEstimateType: true,
+    gasFeeEstimates: true,
+    gasFeeEstimatesByChainId: true,
+    nonRPCGasFeeApisDisabled: false,
+  },
+  KeyringController: {
+    isUnlocked: true,
+    keyrings: false,
+  },
+  LoggingController: {
+    logs: false,
+  },
+  MetamaskNotificationsController: {
```

Comment:
I'm seeing some differences between this constant, and the one being deleted from `setupSentry.js`. Maybe due to merge conflicts? This key is called `NotificationServicesController` for example.

Reply:
I'd recommend copy+pasting this whole constant over from `setupSentry.js` on  `develop` again just to be safe. There were a couple more inconsistencies as well, not just this key.

Reply:
This solved everything e2e related, thanks a bunch!